stdlib/_ast.pyi

```diff
@@ -98,20 +98,20 @@ class Delete(stmt):
 class Assign(stmt):
     if sys.version_info >= (3, 10):
         __match_args__ = ("targets", "value", "type_comment")
-    targets: list[expr]
+    targets: list[Tuple | Name | Attribute | Subscript | Starred]
```

Comment:
Missing List

Reply:
I'd definitely prefer it if we left this one as it is. It feels problematic due to `list` invariance, as we can see from the `mypy_primer` output. The current annotation is more ergonomic, even if it's slightly less precise. The situation will get worse if we add better `__init__` methods for `ast` nodes, as proposed in #8378

We could maybe add a comment:

```suggestion
    # Strictly speaking a target can only be a Tuple, Name, Attribute, List, Subscript, or Starred,
    # But due to `list` invariance, that's a pretty annoying annotation to use here
    targets: list[expr]
```

Reply:
OK I will revert this change. I agree with @AlexWaygood, the current state is better from ergonomic standpoint. 

Btw, as I am working a lot with `ast` module lately (writing something similar to https://www.archunit.org/ but for Python), I can take on the #8378 after this one is done and in general I'd like to contribute to improvement of typing for `ast` module.